src/mongocxx/read_preference.cpp

```diff
@@ -115,7 +115,7 @@ stdx::optional<std::chrono::seconds> read_preference::max_staleness() const {
 }
 
 read_preference& read_preference::hedge(bsoncxx::document::view_or_value hedge) {
-    libbson::scoped_bson_t hedge_bson{hedge.view()};
+    libbson::scoped_bson_t hedge_bson{std::move(hedge.view())};
```

Comment:
so, no need to take a view here, just `std::move(hedge)` in.

Reply:
Ah right, I need to refresh a bit...